app/graphql/types/query_type.rb

```diff
@@ -442,4 +450,21 @@ def reports_by_status(statuses: nil, sort: [{ on: :created_at, direction: :asc }
       sort: sort
     }, statuses)
   end
+
+  field :find_block_by_id, Types::Block, null: true do
+    description 'Find a single Block by ID'
+    argument :id, ID, required: true
+  end
+
+  def find_block_by_id(id:)
+    Loaders::RecordLoader.for(::Block, token: context[:token]).load(id)
+  end
+
```

Comment:
Is this actually useful for any purpose? We should only add fields that we know will be useful, because it's a lot easier to add something than remove it 😉

Reply:
When I first added this I thought about some use (that unfortunately i don't remember atm).
But i agree that now it does not have purpose or use, I'll remove it!